src/js/view/controls/controls.js

```diff
@@ -126,9 +126,10 @@ export default class Controls {
 
         // Settings Menu
         let lastState = null;
-        const visibilityChangeHandler = (visible, isKeyEvent) => {
+        const visibilityChangeHandler = (visible, evt = {}) => {
             const state = model.get('state');
             const settingsInteraction = { reason: 'settingsInteraction' };
+            const isKeyEvent = evt.sourceEvent ? evt.sourceEvent.type === 'keydown' : evt.type === 'keydown';
```

Comment:
Here is a short replacement for the default parameter that also removes duplication in your ternary statement
`const isKeyEvent = (evt && evt.sourceEvent || evt || {}).type === 'keydown';`